cmd/server/api/router_test.go

```diff
@@ -0,0 +1,69 @@
+package api_test
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/freeverseio/laos-universal-node/cmd/server/api"
+	"github.com/freeverseio/laos-universal-node/cmd/server/api/mock"
+	stateMock "github.com/freeverseio/laos-universal-node/internal/platform/state/mock"
+	"github.com/gorilla/mux"
+	"go.uber.org/mock/gomock"
+)
+
+func TestCORS(t *testing.T) {
+	mockCtrl := gomock.NewController(t)
```

Comment:
Add `t.Parallel()`.

If you plan on doing the typical table test (`for _, tc := range tests { tc.Run(...)`), don't forget to add `t.Parallel` inside the subtest too.

Also if you plan on doing that, add the property `name` to the `tests` struct